regex-assembly/942370.ra

```diff
@@ -35,17 +35,17 @@
     ##!=>
   ##!<
   [^\w\s?]+\s*?[^\w\s]+\s*?{{quotes}}
-  [^\w\s]+\s*?[\W\d].*?#
-  [^\w\s]+\s*?[\W\d].*?--
+  [^\w\s]+\s*?[\W\d][^#]*#
+  [^\w\s]+\s*?[\W\d][^-]*--
```

Comment:
Doesn't work. With this I can bypass detection by finding a way to use a single `-` before the comment marker `--`.